pyiron_atomistics/lammps/base.py

```diff
@@ -61,6 +61,28 @@ def __init__(self, project, job_name):
         self._prism = None
         s.publication_add(self.publication)
 
+    @property
+    def units(self):
+        """
+        Type of LAMMPS units used in the calculations. Can be either of 'metal', 'real', 'si', 'cgs', and 'lj'
+
+        Returns:
+            str: Type of LAMMPS unit
+        """
+        if self.input.control["units"] is not None:
+            return self.input.control["units"]
+        else:
+            # Default to metal units
+            return "metal"
+
+    @units.setter
+    def units(self, val):
+        allowed_types = ['metal', 'real', 'si', 'cgs', 'lj']
```

Comment:
Having the static list defined here is dangerous, as we might forget to update it later. Can you use `LAMMPS_UNIT_CONVERSIONS.keys()` ?